docs/static/event-api.asciidoc

```diff
@@ -1,31 +1,43 @@
 [[event-api]]
 === Event API
 
-This section is targeted for plugin developers and users of Logstash's Ruby filter. Below we document recent 
-changes (starting with version 5.0) in the way users have been accessing Logstash's event based data in 
-custom plugins and in the Ruby filter. Note that <<event-dependent-configuration>> 
-data flow in Logstash's config files -- using <<logstash-config-field-references>> -- is 
-not affected by this change, and will continue to use existing syntax.
+This section is targeted for plugin developers and users of {ls}'s Ruby filter.
 
-[float]
-==== Event Object
+Event is the main object that encapsulates data flow internally in {ls} and provides an API for the plugin developers to interact with the event's content.
+Typically, this API is used in plugins and in a Ruby filter to retrieve data and use it for transformations.
+Event objects contain the original data sent to {ls} and any additional fields created during {ls}'s filter stages.
 
-Event is the main object that encapsulates data flow internally in Logstash and provides an API for the plugin 
-developers to interact with the event's content. Typically, this API is used in plugins and in a Ruby filter to 
-retrieve data and use it for transformations. Event object contains the original data sent to Logstash and any additional 
-fields created during Logstash's filter stages.
+[[event-api-event-factory]]
+==== Creating new Events with `EventFactory`
 
-In 5.0, we've re-implemented the Event class and its supporting classes in pure Java. Since Event is a critical component 
-in data processing,  a rewrite in Java improves performance and provides efficient serialization when storing data on disk. For the most part, this change aims at keeping backward compatibility and is transparent to the users. To this extent we've updated and published most of the plugins in Logstash's ecosystem to adhere to the new API changes. However, if you are maintaining a custom plugin, or have a Ruby filter, this change will affect you. The aim of this guide is to describe the new API and provide examples to migrate to the new changes.
+Beginning with {ls} 7.14, the `EventFactory` is the preferred method of creating Events.
```

Comment:
:cupid: thank you for documenting `event_factory` - managed to completely ignore the docs part :reminder_ribbon: